doc/source/ray-air/getting-started.rst

```diff
@@ -44,31 +44,44 @@ Trainer
 .. autoclass:: ray.ml.trainer.Trainer
     :members:
 
-.. automodule:: ray.ml.train.integrations.xgboost
+
+Predictors
+~~~~~~~~~~
+
+.. autoclass:: ray.ml.predictor.Predictor
+    :members:
+
+.. autoclass:: ray.ml.batch_predictor.BatchPredictor
```

Comment:
why are predictor and batch_predictor in 2 separate models?

Reply:
I think mostly to separate the interface from an implementation. It was in the same module in a prior iteration of the original PR, cc @ericl who suggested the separation

Reply:
I think these are separate enough in use cases two modules adds to clarity.